scripts/zones/Northern_San_dOria/IDs.lua

```diff
@@ -118,6 +118,18 @@ zones[xi.zone.NORTHERN_SAN_DORIA] =
         HERE_TAKE_THIS                = 13079, -- Here, take this...
         IF_YOU_WEAR_THIS              = 13080, -- If you put this on and walk around, something...unexpected might happen...
         THANK_YOU                     = 13081, -- Thank you...
+        EGG_HUNT_REWARD1              = 13116, -- Egg-cellent! Here's your prize, kupo!
```

Comment:
Given that these message IDs are all sequential, we could instead have `EGG_HUNT_OFFSET` and all the messages could use  `ID.text.EGG_HUNT_OFFSET + 2` etc. 

The comments also have to match egg-xactly what is listed by UpdateExtractor, otherwise monthly updates will break: https://github.com/LandSandBoat/UpdateExtractor/blob/main/out/scripts/zones/Northern_San_dOria/Text.lua.raw.txt#L12275

Reply:
Note that update extractor texts havent been updated since Jan, so they're likely quite out of date